crates/superposition_types/src/webhook.rs

```diff
@@ -43,9 +50,17 @@ pub enum WebhookEvent {
     ExperimentConcluded,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct EventInfo {
```

Comment:
```suggestion
pub struct WebhookEventInfo {
```

We may have other system events in the future, so wanted to namespace this now